### DISCUSSION TITLE
More contrast in dark mode - Currently hard to separate read/unread emails

### DISCUSSION THREAD

**Discussion by Tailsproxy** [tutao/tutanota#5486] [Ideas]
Hi

I love your service and I see you have all been busy at work, your recent changes to the app have been much appreciated, especially aesthetically. 

One issue with the app and website however is that the dark mode is very hard to use due to the low amount of contrast between read and unread emails. There is a small green dot next to the email but when glancing at my inbox it is not immediately obvious an email is read as the grey of read emails is too close to the white text of unread emails. It does not feel natural to use and requires me to focus more than when I use the default light mode which does not suffer from the same issue due to its white background.

I feel like it would possibly be a simple fix to increase the difference in contrast between the read/unread content and or the dark background. Maybe a pure black mode could help. 

I understand you all have a lot to work on currently and just updated the appearance of the app, but it would be nice to see a more user friendly dark mode.

**Comment by charlag** [comment]
Hi
It is a difficult issue because dark mode usually goes hand-in-hand with more subtle colors and lower contrast, otherwise bright elements "burn" on the dark background.
Generally the contrast in the list is 13:90 which is AAA by WCAG and the dot you mentioned also exists there for assistance. I think it is generally in the dark mode that it would be harder to spot bolder text.

**Reply by Tailsproxy** [reply]
I can understand dark mode usually involves more subtle colours. It however feels like the separation between the read and unread email is very small to the point that for me, I often can't tell the two apart. As mentioned, you do have that helpful green dot but it is not as intuitive when you glance at your inbox, it is quite a small target and easy to miss. 

When I compare Tutanota's dark mode to Proton Mail for example, in Proton I can immediately see a separation between read and unread mail as there is a bigger delta between the white of new mail and grey of read mail. Overall the greyed out mail appears much darker compared to the unread and thus it is easy to visually filter through the inbox. 

To avoid overly bright elements or to much contrast, do you think it would be possible to provide a darker grey for read mail? This would still preserve the muted tone of the dark mode. 

Thank you for taking the time to reply to my suggestion, I understand this is not a huge issue.

**Reply by charlag** [reply]
Hi
Thank you for the ideas. We will come back to it as some point and will try to come up with a good solution.
